### ISSUE TITLE
Request: "Keyboard shortcuts" section in elpy-config

### ISSUE THREAD

**Issue by spacecowboy** [jorgenschaefer/elpy#502]
I don't see an obvious way to override the keyboard shortcuts defined by elpy. What I want to do specifically is unbind the M-left, M-right, M-up, and M-down commands. I use M-left and M-right to navigate text (jump word by word) quickly all the time.

As things are with shortcut keys, you don't realize that you use something until you can't anymore.

**Comment by jorgenschaefer** [comment]
Elpy defines its keys in `elpy-mode-map`. You can use `define-key` to change them there:

```Lisp
(load "elpy")
(define-key elpy-mode-map (kbd "<M-left>") nil)
...
```

Does this do what you want? :-)

**Comment by spacecowboy** [comment]
Yes thank you. It did the trick nicely. Added lines like that in my __init.el__. Though in retrospect, it seems I should have remembered "mode-maps"... Maybe a section in elpy-config is overkill, perhaps only a mention of "see elpy-mode-map" somewhere in the docs would be sufficient.
